zubhub_frontend/zubhub/src/api/api.js

```diff
@@ -859,11 +810,11 @@ class API {
   };
 
   /**
-  * @method getActivity
-  * @author Yaya Mamoudou <yayamamoudou0@gmail.com>
-  *
-  * @todo - describe method's signature
-  */
+   * @method getActivity
```

Comment:
Why are we adding spaces here?

Reply:
I think we should stick to the formatting convention being followed in the code for now instead of adding new linting rules. Changes like these could inflate the PR and make it difficult to review the actual logical changes. 

That said, we can discuss and establish linting rules and eventually add eslint to the project so that everyone has the same formatting standards.

Reply:
Until then, @coderatomy you can use 'save without formatting' option in vscode to avoid changing the lines you haven't touched

Reply:
> Until then, @coderatomy you can use 'save without formatting' option in vscode to avoid changing the lines you haven't touched

The project currently utilizes a Prettier configuration specified in the .prettierrc.yaml file. However, the existing configuration might not be comprehensive or optimal. If we consistently adhere to this configuration, it would mitigate problems associated with incorrect code linting and formatting. Failure to activate the `format on save` feature bring up such issues, allowing improperly formatted code to persist.
Just a clarification @brrkrmn, every file here has a change. That's why I had to save